e2e/test/helpers/RetryOperationHelper.cs

```diff
@@ -51,5 +52,43 @@ public static async Task RetryOperationsAsync(Func<Task> asyncOperation, IRetryP
             }
             while (shouldRetry);
         }
+
+        /// <summary>
+        /// Rety an async operation based on the retry strategy supplied.
+        /// </summary>
+        /// <param name="asyncOperation">The async operation to be retried.</param>
+        /// <param name="retryPolicy">The retry policy to be applied.</param>
+        /// <param name="retryableStatusCodes">The errors to be retried on.</param>
+        /// <param name="logger">The <see cref="MsTestLogger"/> instance to be used.</param>
+        /// <returns></returns>
+        public static async Task RetryOperationsAsync1(Func<Task> asyncOperation, IRetryPolicy retryPolicy, HashSet<IotHubStatusCode> retryableStatusCodes, MsTestLogger logger)
```

Comment:
Why a suffix of 1?

Reply:
The original RetryOperationsAsync() is still used by DPS clients E2E tests, which should be replaced by this method after we re-work the exception thrown by DPS clients.